spec/models/hearing_spec.rb

```diff
@@ -5,69 +5,76 @@
 
   let(:hearing) { described_class.new(body: "{}") }
 
-  describe "validations" do
-    it { is_expected.to validate_presence_of(:body) }
-  end
+  it { is_expected.to validate_presence_of(:body) }
```

Comment:
Would this be your recommended way to do this outside of any describe block? It is like this in a few places. We can put it on a backlog to change at some point if this is preferred.